packages/expo-updates/e2e/setup/project.js

```diff
@@ -524,8 +555,8 @@ async function initAsync(
   return projectRoot;
 }
 
-async function setupE2EAppAsync(projectRoot, localCliBin, repoRoot) {
-  await copyCommonFixturesToProject(projectRoot, 'App.tsx', repoRoot);
+async function setupE2EAppAsync(projectRoot, localCliBin, repoRoot, isTV = false) {
```

Comment:
prefer named arguments for booleans (so it's not a naked `true`/`false` at callsite)
```suggestion
async function setupE2EAppAsync(projectRoot, localCliBin, repoRoot, { isTV } = { isTV: false }) {
```
(I haven't written JS in a long time so I have no idea if this is valid syntax)

Reply:
Sure I can rewrite the function args, either syntax is fine in JS